### PR TITLE
Implement basic parsing for client requests, as well as a way to schedule those commands

### LINKED ISSUE #32: Properly handle AI-client requests.

### LINKED ISSUE #33: Create functions to send notifications to AI-clients.

### PR DESCRIPTION
**Pull request by nils-mathieu**
Fixes #32 
Fixes #33

### CODE REVIEW

**Comment by nils-mathieu** [comment]
not sure why CI failed, will look into it later

**Comment by Eviber** [comment]
C'était quoi le souci du coup ?

**Comment by Eviber** [comment]
C'est bon pour moi, je laisse Tiago review vu que de toute façon on a le temps. :)

File: crates/server/src/main.rs [hunk 1]
```diff
@@ -145,10 +168,37 @@ async fn try_handle_connection(conn: ft::File) -> ft::Result<()> {
     client.send_raw(b"BIENVENUE\n").await?;
     let team_name = client.recv_line().await?;
 
-    ft_log::trace!(
-        "received team name: `{}`",
-        core::str::from_utf8(team_name).unwrap_or("<invalid-utf8>")
-    );
+    if team_name == b"GRAPHIC" {
+        ft_log::trace!("client #{id} is a graphical monitor");
+        todo!();
```
Comment (Ragarnoy) [inline crates/server/src/main.rs:173]: Usually todo's are supposed to be things that are to be changed before a commit, so they are not supposed to be pushed. For unimplemented code use instead `unimplemented!()'

**Review by Ragarnoy — APPROVED** [review]
Une modif mineure sinon rien à dire

**Review by Eviber — APPROVED** [review]

File: crates/server/src/main.rs (hunk 1 above)
Comment (nils-mathieu) [reply crates/server/src/main.rs:173]: trueeeee

**Pull request merged**